Application/EdFi.Common/EdFi.Common.csproj

```diff
@@ -13,9 +13,9 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="3.1.3" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="5.0.0" />
```

Comment:
@jamessmckay In the JIRA ticket you wrote there should be no major version upgrades. Does this constitute a major version upgrade?

Reply:
I limited to  minor upgrades . thanks